uqcsbot/scripts/meme.py

```diff
@@ -0,0 +1,308 @@
+from uqcsbot import bot, Command
+import re
+from uqcsbot.util.status_reacts import loading_status, success_status
+
+API_URL = "https://memegen.link/"
+# Many different characters need to be replaced in order to work in url format
+# See the API_URL for details
+REPLACEMENTS = {'_': '__', ' ': '_', r'\"': "''", '-': '--', '?': '~q', '%': '~p', '#': '~h', '/': '~s'}
```

Comment:
Do we want something to do with https://docs.python.org/3/library/urllib.parse.html#url-quoting here?

Reply:
Does that allow custom changes? For instance the api expects forward slashes to be encoded in the url as: ~s

Reply:
I think using a custom solution is fine tbh, plus it's all explained at memegen.link